catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -511,6 +551,15 @@ private void writeToZip(/*Mutable,IO*/ ZipFile zipFile, Result result) {
     }
   }
 
+  private void writeDataToZip(ZipOutputStream zipOutputStream, InputStream inputStream)
```

Comment:
✏️ This could be replaced with `IOUtils.copy()`

Reply:
👍